tests/anthropic/test_anthropic_autolog.py

```diff
@@ -44,6 +44,12 @@ def test_messages_autolog():
     assert span.name == "Messages.create"
     assert span.span_type == SpanType.CHAT_MODEL
     assert span.inputs == DUMMY_CREATE_MESSAGE_REQUEST
+    original_usage = span.outputs["usage"]
+    # Only keep input_tokens / output_tokens fields in usage dict.
+    span.outputs["usage"] = {
+        "input_tokens": original_usage["input_tokens"],
+        "output_tokens": original_usage["output_tokens"],
+    }
```

Comment:
span.outputs["usage"] = {key: span.outputs["usage"][key] for key in ["input_tokens", "output_tokens"]}